processor/src/main/java/org/mapstruct/ap/internal/model/Field.java

```diff
@@ -128,4 +130,12 @@ public boolean equals(Object obj) {
             (other.variableName != null) : !this.variableName.equals( other.variableName ) );
     }
 
+    public static List<String> getFieldNames(List<Field> fields) {
+        List<String> names = new ArrayList<String>( fields.size() );
```

Comment:
Are we on Java 8 already? If so, it could be nicely done using streams. No strong need to change of course.

Reply:
no.. we aren't unfortunately..